test/run/ch-convert.bats

```diff
@@ -116,7 +116,8 @@ compare-ls () {
     cd "$1" || exit  # to make -path reasonable
       find . -mindepth 1 \
               \(    -path ./.dockerenv \
-                 -o -path ./ch \) -prune \
+                 -o -path ./ch  \
+	         -o -path ./run \) -prune \
```

Comment:
This overreaches a bit (excludes all of `/run`, when we just want `/run` itself and `/run/.containerenv`, but that's probably fine.